### PR TITLE
Fixed RA parabomb/paradrop support power tooltips

### PR DESCRIPTION
**Pull request by penev92**
The AFLD's AirstrikePower@parabombs' description wasn't updated to the latest balance patch (3 Badgers -> 1 Badger)

As reported by @deniz1a on Discord.

### CODE REVIEW

**Review by PunkPun — DISMISSED** [review]
LGTM

File: mods/ra/rules/misc.yaml [hunk 1]
```diff
@@ -346,7 +346,7 @@ powerproxy.parabombs:
 	AirstrikePower:
 		Icon: parabombs
 		Name: Parabombs (Single Use)
-		Description: A Badger drops a load of parachuted bombs on your target.
+		Description: A Badger drops a load of parachuted bombs\nat a selected location.
```
Comment (abcdefg30) [inline mods/ra/rules/misc.yaml]:
```suggestion
		Description: A Badger drops a load of parachuted bombs\nat the selected location.
```
to emphasize that it's not any location, but the one you select.

Comment (penev92) [reply mods/ra/rules/misc.yaml]: Done

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/58213205db195163e1591e4caa7332c1c23a58dc...c0eb2678d49b5b311ad01092d16f32f13d6a7f02)

https://github.com/OpenRA/OpenRA/commit/ea221f9bfbf6c02277fe8fdb9646237180e91af3

**Pull request merged**